internal/query/projection/idp_template_test.go

```diff
@@ -410,6 +410,270 @@ func TestIDPTemplateProjection_reducesOAuth(t *testing.T) {
 	}
 }
 
+func TestIDPTemplateProjection_reducesAzureAD(t *testing.T) {
+	type args struct {
+		event func(t *testing.T) eventstore.Event
+	}
+	tests := []struct {
+		name   string
+		args   args
+		reduce func(event eventstore.Event) (*handler.Statement, error)
+		want   wantReduce
+	}{
+		{
+			name: "instance reduceAzureADIDPAdded",
```

Comment:
would add a minimal test as some attributes can be null in the database, at least for either org or instance